### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by progrium** [progrium/topframe#8] [General]
## Changelog

fac6f06 add release automation
53261d5 fix redundant description
209e910 make minor tweak to readme bullets
a7bfb55 add demo gif to readme and update usage instructions
7348841 drop subcommands in favor of simpler flags
dcf0e51 add docs to stocks script
6e4939a add minor fixes to readme
30aeb4c add minor fixes to readme
97a53bc update install instructions
fce390d add final demo index.html, and made demo files serve from binary
9450b9a final touches
55212e4 hide go report card for now
752ceee near complete 0.2.0 with subcommands and scripting support
bf220a4 more docs
07a9b5f improved default index, more readme
9eafa0d embedding in package level variables since function level embeds were removed
13ceb65 initial goreleaser config
f569dd2 Create FUNDING.yml
8a492a5 initial working 0.2.0 version
23003b7 readme progress
27d545a initial readme draft
36341e3 Initial commit

<hr /><em>This discussion was created from the release <a href='https://github.com/progrium/topframe/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by salsepulveda** [comment]
Hlw can you help me

**Reply by progrium** [reply]
ask your question to find out

**Reply by salsepulveda** [reply]
I wanted to add a community info to my read me and how can i add a payment method

**Reply by progrium** [reply]
I'm sorry that doesn't make any sense to me. Are you talking about the badges on the readme here? Just look at the source code.

**Reply by salsepulveda** [reply]
I have the source code  with me but i dont understand how to make it work here

**Reply by progrium** [reply]
I'm not going to be able to help you.
